spec/features/conference_registration_spec.rb

```diff
@@ -49,5 +66,15 @@
         expect(conference.user_registered?(participant)).to be(true)
       end
     end
+
+    context 'registration is closed' do
+      let(:conference_with_closed_registration) { create(:conference, registration_period: create(:registration_period)) }
+
+      scenario 'registers for a conference', feature: true do
+        visit new_conference_conference_registrations_path(conference_with_closed_registration.short_title)
+        expect(current_path).to eq(root_path)
+        expect(flash).to eq 'You are not authorized to access this page.'
```

Comment:
Right, in that case why do we still have code
https://github.com/sonalkr132/osem/blob/registration_period/app/controllers/conference_registrations_controller.rb#L22 ?